src/components/LoanCards/FavoriteStar.vue

```diff
@@ -26,10 +29,58 @@ export default {
 	},
 	methods: {
 		toggleFavorite() {
-			this.$emit('favorite-toggled');
+			if (this.isFavorite === false) {
```

Comment:
Spoke with @BoulderBrains about incorporating this into GridLoanCard and LoanCardController instead.